modules/cli/test-scripts/streams-tests.sh

```diff
@@ -0,0 +1,250 @@
+    local bootstrap=$2
+    
+    local desc_prop_name="test.stream.${stream_name}.description"
+    local maven_prop_name="test.stream.${stream_name}.repo"
+    local testCatalog_prop_name="test.stream.${stream_name}.location"
+    local obr_prop_name="test.stream.${stream_name}.obr"
+
+    # Define property settings with their commands
+    local cmd1="${BINARY_LOCATION} properties set --namespace framework \
+    --name $desc_prop_name \
+    --value Galasa \
+    --bootstrap $bootstrap \
+    --log -"
+
+    local cmd2="${BINARY_LOCATION} properties set --namespace framework \
+    --name $maven_prop_name \
+    --value http://points-to-my-maven-repo.example.org \
+    --bootstrap $bootstrap \
+    --log -"
+
+    local cmd3="${BINARY_LOCATION} properties set --namespace framework \
+    --name $testCatalog_prop_name \
+    --value http://points-to-my-test-catalog.example.org \
+    --bootstrap $bootstrap \
+    --log -"
+
+    local cmd4="${BINARY_LOCATION} properties set --namespace framework \
+    --name $obr_prop_name \
+    --value mvn:myorg/myartifact/0.0.1/obr \
+    --bootstrap $bootstrap \
+    --log -"
+
+    info "Command is: $cmd1"
+    info "Command is: $cmd2"
+    info "Command is: $cmd3"
+    info "Command is: $cmd4"
+
+    $cmd1
+    $cmd2
+    $cmd3
+    $cmd4
+    local rc=$?
+
+    # We expect a return code of 0 because this is a properly formed properties set command.
+    if [[ "${rc}" != "0" ]]; then 
+        error "Failed to create property with name and value used."
+        return 1
+    fi
+    
+    return 0
+}
+
+#-----------------------------------------------------------------------------------------
+# Tests
+#----------------------------------------------------------------------------------------- 
+function streams_create {
+    h2 "Performing properties set with name and value parameter used: create..."
+
+    set -o pipefail # Fail everything if anything in the pipeline fails. Else we are just checking the 'tee' return code.
+    
+    stream_name="mystream"
+    configure_stream "$stream_name" "$bootstrap"
+    rc=$?
+    
+    if [[ "${rc}" != "0" ]]; then 
+        error "Failed to configure stream properties."
+        exit 1
+    fi
+
+    # check that stream has been created
+    cmd="${BINARY_LOCATION} streams get \
+    --name $stream_name \
+    --bootstrap $bootstrap \
+    --log -"
+
+    info "Command is: $cmd"
+
+    output_file="$ORIGINAL_DIR/temp/streams-get-output.txt"
+    $cmd | tee $output_file
+    if [[ "${rc}" != "0" ]]; then 
+        error "Failed to get stream with name used: command failed."
+        exit 1
+    fi
+
+    # Check that the previous properties set created a stream property
+    cat $output_file | grep "$stream_name" -q
+
+    rc=$?
+    # We expect a return code of 0 because this is a properly formed streams get command.
+    if [[ "${rc}" != "0" ]]; then 
+        error "Failed to create stream with name and value used."
+        exit 1
+    fi
```

Comment:
Failed to get stream? Or something saying that the output is incorrect/missing the stream name?